index.rst

```diff
@@ -73,23 +73,25 @@ EditorConfig organization.
 File Format
 ===========
 
+.. versionchanged:: 0.16.1
+
 EditorConfig files are in an INI-like file format.
-In an EditorConfig file, all beginning whitespace on each line is considered
-irrelevant. Each line must be one of the following:
+In an EditorConfig file, all beginning whitespace on each line is ignored.
+Each line must be one of the following, once leading whitespace is removed
+(and ignoring any trailing line separator):
 
-- Blank: contains only whitespace characters.
+- Blank: contains nothing.
```

Comment:
Maybe

```suggestion
- Blank: contains nothing or only whitespace characters.
```
?

Reply:
Indeed, I scratched my head a bit about that as well.  The line just before the bullets says "once leading whitespace is removed", so there would be nothing left.  I thought that was cleaner than adding "may contain leading whitespace" to each bullet.

How about moving blanks out of the bullets?  Something like:
```diff
+When reading an EditorConfig file, the following are ignored:
+ - blank lines (empty, or nothing but whitespace)
+ - all beginning whitespace on each line.
+Each non-blank line must be one of the following, once leading whitespace is removed
+(and ignoring any trailing line separator):
- - Blank: contains only whitespace characters
  - Comment: ...
```